microprofile/microprofile-fault-tolerance-2.0/src/test/java/org/wildfly/swarm/ts/microprofile/fault/tolerance/v20/PriorityOrderTest.java

```diff
@@ -0,0 +1,64 @@
+package org.wildfly.swarm.ts.microprofile.fault.tolerance.v20;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+
+import org.apache.http.client.fluent.Request;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.swarm.arquillian.DefaultDeployment;
+import org.wildfly.swarm.ts.microprofile.fault.tolerance.v20.priority.AfterFT;
+import org.wildfly.swarm.ts.microprofile.fault.tolerance.v20.priority.BeforeFT;
+import org.wildfly.swarm.ts.microprofile.fault.tolerance.v20.priority.InterceptorsContext;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+// Clarify how the Fault Tolerance annotations interact with other application defined interceptors (#313)
```

Comment:
Do you just cite MP FT issue names here? I don't think that adds value. If you could rephrase in the class javadoc, that would be great. Links to MP FT issues are optional IMHO :-)

Reply:
forgotten comments